packages/devextreme/testing/tests/DevExpress.ui.widgets.editors/selectBox.tests.js

```diff
@@ -6101,6 +6101,18 @@ if(devices.real().deviceType === 'desktop') {
 
             assert.strictEqual(helper.$widget.find(`.${TEXTEDITOR_INPUT_CLASS}`).attr('aria-autocomplete'), 'list');
         });
+
+        QUnit.test('aria-haspopup and aria-autocomplete attributes should exist when fieldTemplate is rendered (T1230696)', function(assert) {
+            helper.createWidget({
+                fieldTemplate: () => {
+                    return $('<div>').dxTextBox();
+                },
+            });
+
+            assert.ok(helper.$widget.find(`.${TEXTEDITOR_INPUT_CLASS}`).attr('aria-haspopup'), true);
```

Comment:
These assertions and comments lack informativeness. What values are stored in these attributes? Can we use the strictEqual function here and provide corresponding comments for each attribute?